backend/config/database.php

```diff
@@ -43,6 +43,14 @@
             'foreign_key_constraints' => env('DB_FOREIGN_KEYS', true),
         ],
 
+        'sqlite_test' => [
+            'driver' => 'sqlite',
+            'url' => env('DATABASE_URL'),
+            'database' => env('DB_DATABASE', database_path('database_test.sqlite')),
+            'prefix' => '',
+            'foreign_key_constraints' => env('DB_FOREIGN_KEYS', true),
+        ],
+
```

Comment:
what happened here?

Reply:
I added a sqlite file for database testing in other not for the phpunit tests to interfere with the main database